trendmicro_deepsecurity/icon_trendmicro_deepsecurity/util/shared.py

```diff
@@ -0,0 +1,34 @@
+
+from komand.exceptions import PluginException
+
+def tryJSON(response):
+    """
+    Try to convert response data to JSON
+    """
+
+    try:
+        response_data = response.json()
+    except json.decoder.JSONDecodeError:
+        raise PluginException(
+                            cause="Received an unexpected response from Deep Security",
+                            assistance="(non-JSON or no response was received).",
+                            data=response.text
+                            )
+    return response_data
+
+def checkResponse(response):
```

Comment:
Can you utilize type hinting and annotations?

Example:
`def add_nums(first: int, second: int) -> int`

Reply:
done